### PR TITLE
adhere to basic robocop style

### PR DESCRIPTION
**Pull request by tybug**
includes a few of my own manual corrections, but the bulk is from `rubocop --auto-correct`, which [claims to only make safe corrections](https://docs.rubocop.org/rubocop/usage/auto_correct.html#safe-auto-correct). No idea how accurate that claim is but I went through all of the changes briefly and they all seemed reasonable. 

We'll want to test this pretty thoroughly even so I think. Maybe a good excuse to write more tests? Not sure how comprehensive the current suite is.

### CODE REVIEW

File: app/scrapers/bus_routes_scraper.rb [hunk 1]
```diff
@@ -11,54 +11,58 @@
...
+    {
+      direction_id: e['tag'],
+      title: e['title'],
+      stops: e['stop'].map do |stop|
+               begin
+                                      stop['tag']
+               rescue StandardError
+                 e['stop']
+                                    end
+             end
```
Comment (tybug) [inline app/scrapers/bus_routes_scraper.rb:53]: this correction definitely needs to be fixed properly, very ugly as is

File: app/controllers/majors_controller.rb [hunk 2]
```diff
@@ -8,26 +8,26 @@ def self.registered(app)
...
+                version: '1.0.0',
+                docs: 'https://docs.umd.io/majors',
+                endpoints: ['/list']
               }
               json resp
             end
 
             get '/list' do
-              json Major.all.map {|m| m.to_v1}.uniq
+              json Major.all.map { |m| m.to_v1 }.uniq
```
Comment (DonIsaac) [inline app/controllers/majors_controller.rb:20]: why not `&:to_v1`?

File: app/controllers/majors_controller.rb [hunk 3]
```diff
@@ -8,26 +8,26 @@ def self.registered(app)
...
             end
           end
 
           app.namespace '/v0/majors' do
             get do
-              json Major.all.map {|m| m.to_v0}.uniq
+              json Major.all.map { |m| m.to_v0 }.uniq
```
Comment (DonIsaac) [inline app/controllers/majors_controller.rb:26]: why not `&:to_v0`?

File: app/controllers/map_controller.rb [hunk 4]
```diff
@@ -9,22 +9,22 @@ def self.registered(app)
...
+                docs: 'https://docs.umd.io/map',
+                endpoints: ['/buildings', '/buildings/{:building_id}']
               }
               json resp
             end
 
             # get list of all buildings with names and numbers
             get '/buildings' do
-              json Building.all.map {|b| b.to_v1}
+              json Building.all.map { |b| b.to_v1 }
```
Comment (DonIsaac) [inline app/controllers/map_controller.rb:22]: same

File: app/controllers/map_controller.rb [hunk 5]
```diff
@@ -9,22 +9,22 @@ def self.registered(app)
...
             end
 
             # get buildings by building_id or code
             get '/buildings/:building_id' do
-              buildings = get_buildings_by_id(params[:building_id]).map {|b| b.to_v1}
+              buildings = get_buildings_by_id(params[:building_id]).map { |b| b.to_v1 }
```
Comment (DonIsaac) [inline app/controllers/map_controller.rb:27]: same

File: app/helpers/courses_helpers.rb [hunk 6]
```diff
@@ -2,14 +2,13 @@
 module Sinatra
   module UMDIO
     module Helpers
-
       # Generic course helpers
 
       # generalize logic for checking if semester param valid
-      def check_semester app, semester
+      def check_semester(_app, semester)
```
Comment (DonIsaac) [inline app/helpers/courses_helpers.rb:8]: Doesn't look like `_app` is used, should it be removed entirely?

**Review by DonIsaac — CHANGES_REQUESTED** [review]

File: app/controllers/majors_controller.rb (hunk 2 above)
Comment (tybug) [reply app/controllers/majors_controller.rb:20]: definitely should be, that rubocop rule must not be safe and so was not applied automatically. Feel free to make the changes, there were just a lot of them and I didn't want to do them all manually just yet (though we will need to do so eventually)

File: app/helpers/courses_helpers.rb (hunk 6 above)
Comment (tybug) [reply app/helpers/courses_helpers.rb:8]: I think so, I wasn't 100% comfortable removing parameters before understanding why they were there in the first place, but should be fine to remove

**Comment by McIntireEvan** [comment]
Skimmed and overall looks fine, seeing as it's all mostly automatic. Let's do follow-up changes in a separate PR - something of this size is really unwieldy to review already, don't want changes that need more thought getting lost in a sea of spacing and quote changing

**Review by DonIsaac — APPROVED** [review]

**Comment by DonIsaac** [comment]
looks 👍

**Review by DonIsaac — APPROVED** [review]
👍

**Comment by DonIsaac** [comment]
@tybug please merge this is this is good to go

**Pull request merged**